src/Raven.Studio/typescript/viewmodels/database/tasks/ongoingTasks.ts

```diff
@@ -369,7 +374,8 @@ class ongoingTasks extends viewModelBase {
                     default:
                         return taskType;
                 }
-            }));
+            })
+            .sort((a, b) => { return this.tasksOrderForUI[a] - this.tasksOrderForUI[b] }));
```

Comment:
does this sort stable? https://www.geeksforgeeks.org/stability-in-sorting-algorithms/